caseworker/bookmarks/services.py

```
@@ -0,0 +1,67 @@
+import datetime
+import json
+from collections import OrderedDict
+
+from django.urls import reverse
+from django.utils.http import urlencode
+
+from caseworker.users.services import get_gov_user
+from core import client
+
+
+class BookmarkException(Exception):
+    pass
+
+
+def fetch_bookmarks(request):
+    bookmarks = client.get(request, "/bookmarks/").json()
```

Would suggest some response code checking here so that we can fail early